build-info/dotnet/roslyn/netcore1.0/Latest_Packages.txt

```diff
@@ -1,9 +1,9 @@
-Microsoft.CodeAnalysis.Common 1.3.0-beta1-20160610-04
-Microsoft.CodeAnalysis.Workspaces.Common 1.3.0-beta1-20160610-04
-Microsoft.CodeAnalysis.CSharp 1.3.0-beta1-20160610-04
-Microsoft.CodeAnalysis.CSharp.Workspaces 1.3.0-beta1-20160610-04
-Microsoft.CodeAnalysis.VisualBasic 1.3.0-beta1-20160610-04
-Microsoft.CodeAnalysis.VisualBasic.Workspaces 1.3.0-beta1-20160610-04
-Microsoft.CodeAnalysis.Compilers 1.3.0-beta1-20160610-04
-Microsoft.Net.Compilers.netcore 1.3.0-beta1-20160610-04
-Microsoft.DiaSymReader.PortablePdb 1.1.0-beta1-60610-04
+Microsoft.CodeAnalysis.Common 1.3.0
+Microsoft.CodeAnalysis.Workspaces.Common 1.3.0
+Microsoft.CodeAnalysis.CSharp 1.3.0
+Microsoft.CodeAnalysis.CSharp.Workspaces 1.3.0
+Microsoft.CodeAnalysis.VisualBasic 1.3.0
+Microsoft.CodeAnalysis.VisualBasic.Workspaces 1.3.0
+Microsoft.CodeAnalysis.Compilers 1.3.0
+Microsoft.Net.Compilers.netcore 1.3.0
```

Comment:
Yeah, I was mistaken. We have destkop version of the package which depends on pre-release diasymreader. Our package naming is confusing :(